prisma/seed.ts

```diff
@@ -1,6 +1,12 @@
 import { PrismaClient, Prisma } from "@prisma/client";
 import { hash } from "bcrypt";
-import recommendation from "../backend/recommendation";
+
+import { movies } from "../data/movies.json";
+
+const formattedMovies = movies.map((movie) => {
+  return { ...movie, release_date: new Date(movie.release_date) };
+});
+
```

Comment:
Is this function used for the seed? I can only see the release date as 2023 still hardcoded